### PR TITLE
Removes hats from starting/default uniforms

### PR DESCRIPTION
**Pull request by DreamySkrell**
changes:
  - rscdel: "Removes hats from starting/default uniforms. All the hats can still be selected in the loadout, or found in lockers."
  - rscadd: "Adds uniform hats to relevant jobs' lockers, where missing"

hats begone

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -340,8 +340,13 @@
...
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/head/bridge_crew_cap
+	display_name = "bridge crew cap"
+	path = /obj/item/clothing/head/caphat/bridge_crew
+	allowed_roles = list("Bridge Crew", "Captain", "Executive Officer")
+
 /datum/gear/head/bridge_crew_cap_alt
-	display_name = "san colettish bridge crew cap"
+	display_name = "bridge crew cap, san colettish"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:349]:
```suggestion
	display_name = "bridge crew cap (san colettish)"
```

**Review by SleepyGemmy — APPROVED** [review]
PR looks fine as far as i can see.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Adding them to the lockers is good (and should be moved to a different PR) but won't solve the issue that they will run out after long rounds.

This implementation is rather extreme and I would not like this merged unless it is changed

The goal of this PR, to have default hats not coincide with outfits can be as easily achieved by having a simple loadout toggle rather than the nuclear option of removing ALL the hats from the default uniforms.

**Comment by DreamySkrell** [comment]
> Adding them to the lockers is good (and should be moved to a different PR) but won't solve the issue that they will run out after long rounds.

The goal of this PR is not to solve the issue of gear running out on long rounds, and it should not be. This PR does not affect this issue, and does not make it worse nor better. All the hats still exist, in lockers and/or the loadout. Same as all the other gear, uniforms, alt-uniforms, and everything else.

> This implementation is rather extreme and I would not like this merged unless it is changed

Changed how. Do not use the "changes requested" tag if you disagree with this PR completely.

> The goal of this PR, to have default hats not coincide with outfits can be as easily achieved by having a simple loadout toggle rather than the nuclear option of removing ALL the hats from the default uniforms.

Disagreed. We do not need more microspecific toggles. This is not the nuclear option. All the hats still exist, in lockers and/or the loadout.

**Comment by GMR25** [comment]
A bit unfair for those who already had full loadouts and may not be able to afford the hat to finish their appearance, especially if they latejoined in a round where their slot had been taken before so the clothing in the locker is gone. I truly do not understand the issue with spawning with hats.

**Comment by DreamySkrell** [comment]
> A bit unfair for those who already had full loadouts and may not be able to afford the hat to finish their appearance, 

You could say the same about the alternative uniforms too, or drip in general. In the current system, there is no way to make everyone happy, unfortunately.

> especially if they latejoined in a round where their slot had been taken before so the clothing in the locker is gone.

They're just hats. They still exist. Most people do not wear hats anyways. The probability that someone was in the round before you, and wore the exact same outfit as you that included the hat, is very slim. People leaving the round with actual gear (tools, weapons, armor, etc) is a far more serious issue than hats.

> I truly do not understand the issue with spawning with hats.

The issue is. If you plan your outfit around not wearing a hat, you still have the hat in the loadout preview and when ghosting. It ruins the drip, and is annoying. You can press "hide job gear", but then you lose everything (uniform, boots, bag, radio, and anything else). People want to preview their loadout for some jobs without hats in the way. Most hats also very badly fit skrell or some robots, or are ugly and get very little use.

**Comment by Butterrobber202** [comment]
this not kosher. Esp since Engineers spawn with hats that have mechanical benefits for their job.

**Comment by DreamySkrell** [comment]
> this not kosher. Esp since Engineers spawn with hats that have mechanical benefits for their job.

Not an issue. Engineers will be able to pick hard hats from the loadout or from the lockers.

**Comment by SleepyGemmy** [comment]
> Adding them to the lockers is good (and should be moved to a different PR) but won't solve the issue that they will run out after long rounds.

i think that most if not all role-related hats are in the loadout already.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
ensure that role-related hats exist in the loadout for all roles this PR touches and make them cost zero points. i suggest using the loadout "Faction" tab for this.

only exception is hardhats which should remain as they are, since they offer mechanical advantage.

**Pull request closed without merging**